app/build.gradle

```diff
@@ -23,8 +23,8 @@ android {
         buildTypes {
             debug {
                 buildConfigField "String", "BASE_URL", '"http://gateway.marvel.com/"'
-                buildConfigField "String", "PUBLIC_KEY", marvelPublicKey
-                buildConfigField "String", "PRIVATE_KEY", marvelPrivateKey
+                buildConfigField "String", "PUBLIC_KEY", "\"$marvelPublicKey\""
```

Comment:
i try this in case gradle.properties configuration doesn't exists for don't break build...
but they still breaking build.. 
i will rollback for "" declaration on gradle.properites